src/System.CommandLine/RootCommand.cs

```diff
@@ -7,12 +7,22 @@
 
 namespace System.CommandLine
 {
+    /// <summary>
+    /// A command representing entry point of an application and used to Invoke the correct handlers.
```

Comment:
Not sure the Invoke potion of this is really accurate. This class is not actually responsible for doing the invocation, rather holding the data for parsing/invocation.

```suggestion
    /// A command representing entry point of an application.
```

Reply:
```suggestion
    /// A command representing an application entry point.
```